src/components/app-top-bar.js

```diff
@@ -382,8 +400,37 @@ const AppTopBar = ({ user, tabIndex, onChangeTab, userManager }) => {
                                 return <Tab key={tabName} label={label} />;
                             })}
                         </Tabs>
-                    )}
-                    {studyUuid && (
+                        <Box sx={styles.searchButton}>
+                            <Tooltip
+                                title={
+                                    <FormattedMessage id="equipment_search/label" />
+                                }
+                            >
+                                <Button
+                                    color="inherit"
+                                    size="large"
+                                    onClick={() => setDialogSearchOpen(true)}
+                                >
+                                    <Search />
+                                </Button>
+                            </Tooltip>
+                        </Box>
+                        <Box sx={styles.searchButton}>
+                            <Tooltip
+                                title={
+                                    <FormattedMessage id="equipment_search/label" />
+                                }
+                            >
+                                <IconButton
+                                    color="inherit"
+                                    aria-label="search"
+                                    size="large"
+                                    onClick={() => setDialogSearchOpen(true)}
+                                >
+                                    <Search />
+                                </IconButton>
+                            </Tooltip>
+                        </Box>
```

Comment:
Il y a une différence entre les deux boutons search ?

Reply:
oups, I forget to remove it!

Yes they don't have the same apparence (mainly one is ~sqaure~ retangle and the other is round.